packit_service/worker/build/build_helper.py

```diff
@@ -240,7 +240,9 @@ def _report(
             description=description, state=state, url=url, check_names=check_names,
         )
 
-    def report_status_to_all(self, description: str, state: str, url: str = "") -> None:
+    def report_status_to_all(
+        self, description: str, state: Union[CommitStatus, str], url: str = ""
```

Comment:
It would be nice to not allow `str` and let the `mypy` find the wrong calls...

Reply:
changed